helm/nginx-ingress-controller-app/templates/controller-rolebinding.yaml

```diff
@@ -1,15 +1,21 @@
-kind: RoleBinding
+{{- if .Values.rbac.create -}}
 apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
 metadata:
-  name: {{ include "ingress-nginx.fullname" . }}-ng
```

Comment:
In before: Renaming works, [tested it](https://gigantic.slack.com/archives/C02GLN41UP2/p1670941098436149). :)

Reply:
Can confirm. It works